test/e2e/tests/change-language.spec.js

```diff
@@ -84,6 +84,7 @@ describe('Settings - general tab, validate the change language functionality:',
         );
         assert.equal(isLanguageLabelChanged, true, 'Language did not change');
 
```

Comment:
We are trying to move await from using delays in our tests. I wonder what this test needs to wait for before is correctly selected?

Reply:
In general, yes, but I think it's going to require some large structural changes to driver.js and helpers.js to accomplish this goal.

Reply:
Sure I agree. I will make the changes to wait for the element.

Reply:
Done